scripts/trim_tract_cat.py

```diff
@@ -96,9 +106,11 @@ def make_trim_file(infile, output_file=None, output_dir=None,
 """)
     parser.add_argument('--output_dir', default='./',
                         help='Output directory.  (default: %(default)s))')
+    parser.add_argument('--verbose', default=False, action='store_true')
```

Comment:
`store_true` by itself already means default is `False`
```suggestion
    parser.add_argument('--verbose', action='store_true')
```

Reply:
Thanks, I didn't know that.